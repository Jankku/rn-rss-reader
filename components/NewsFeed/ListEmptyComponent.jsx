import { useTheme } from '@react-navigation/native';
import { Text, StyleSheet } from 'react-native';

function ListEmptyComponent({ text }) {
  const { colors } = useTheme();
  const styles = makeStyles(colors);

  return <Text style={styles.text}>{text}</Text>;
}

const makeStyles = (colors) =>
  StyleSheet.create({
    text: { color: colors.text, fontSize: 16, fontWeight: '700', textAlign: 'center' },
  });

export default ListEmptyComponent;
