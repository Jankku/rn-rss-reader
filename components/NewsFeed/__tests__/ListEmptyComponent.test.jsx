import ListEmptyComponent from '../ListEmptyComponent';
import { render } from '@testing-library/react-native';

describe('Test ListEmptyComponent', () => {
  test('Should match snapshot', () => {
    const { toJSON } = render(<ListEmptyComponent text={'Text'} />);
    expect(toJSON()).toMatchSnapshot();
  });
});
