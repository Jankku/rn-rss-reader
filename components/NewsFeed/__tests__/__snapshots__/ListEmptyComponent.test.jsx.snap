// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ListEmptyComponent Should match snapshot 1`] = `
<Text
  style={
    Object {
      "color": "rgb(28, 28, 30)",
      "fontSize": 16,
      "fontWeight": "700",
      "textAlign": "center",
    }
  }
>
  Text
</Text>
`;
